static/js/home/display-show.js

```diff
@@ -1,4 +1,13 @@
 MEDUSA.home.displayShow = function() { // eslint-disable-line max-lines
+	$('.imdbPlot').click(function() {
+		$(this).prev('span').toggle();
+		if ($(this).html() === "..show less") {
+			$(this).html("..show more");
+		} else {
+			$(this).html("..show less");
+		}
+	});
```

Comment:
Add a call to moveSummaryBackground(); after the if.. else... check here to resize the background bar